docs/index.md

```diff
@@ -850,6 +851,10 @@ Specifies the test-case timeout, defaulting to 2 seconds. To override you may pa
 
 Specify the "slow" test threshold, defaulting to 75ms. Mocha uses this to highlight test-cases that are taking too long.
 
+### `--file <file>`
+
+Add a file you want included first in a test suite. This is useful if you have some generic setup code that must be included within the test suite. The file passed is not effected by any other flags (`--recursive` or `--sort` have no effect). Accepts multiple `--file` flags to include multiple files.
```

Comment:
- [ ] `--recursive` *does* come into play, afaict; so it's only `--sort` that doesn't apply.
- [ ] should be "is not affected" not "is not effected"
- [ ] let's mention that the order in which the flags are specified is relevant, and
- [ ] it can be used in `mocha.opts`

Reply:
I'm not seeing recursive effect. On line 476 it's taking the args passed in and then iterating over those values and applying recursive to them. Am I missing something?